core/variant/variant.h

```diff
@@ -835,11 +835,28 @@ class Variant {
 	static void get_utility_function_list(List<StringName> *r_functions);
 	static int get_utility_function_count();
 
-	//argsVariant call()
+	[[nodiscard]] bool operator==(const Variant &p_variant) const;
+	[[nodiscard]] bool operator<(const Variant &p_variant) const;
+	[[nodiscard]] _ALWAYS_INLINE_ bool operator!=(const Variant &p_variant) const { return !(*this == p_variant); }
```

Comment:
We currently make a distinction between `!=` and `!(==)`. I don't think this makes sense, but I don't think this PR is the right place to change this. I'd keep the "OP_NOT_EQUAL" implementation for now.

Reply:
I'll make this PR exclusively about the disabled operators then. The distinction is dumb, but it can be handled separately.